apps/site/src/app/(main)/portal/@applicant/VerticalTimeline.tsx

```diff
@@ -111,6 +112,17 @@ function VerticalTimeline({ status }: VerticalTimelineProps) {
 				/>
 				Attendance confirmed
 			</li>
+		) : status === PortalStatus.void ? (
+			<li className="flex flex-row items-center border-t">
+				<Image
+					src={XCircle}
+					alt="checked-circle"
+					width={25}
+					height={25}
+					className="m-6 mr-12"
+				/>
+				Can not RSVP at this time.
```

Comment:
Nitpick: Maybe "No RSVP indicated' could be a better status message?

Reply:
I agree. The change is made.